app/components/forms/login-form.js

```diff
@@ -84,27 +84,10 @@ export default Component.extend(FormMixin, {
     async auth(provider) {
       try {
         if (provider === 'facebook') {
-          this.get('torii').open('facebook').then(authData => {
-            this.get('loader').load(`/auth/oauth/login/${  provider  }/${  authData.authorizationCode  }/?redirect_uri=${  authData.redirectUri}`)
-              .then(async response => {
-                let credentials = {
-                  'identification' : response.email,
-                  'password'       : response.facebook_login_hash
-                };
-
-                let authenticator = 'authenticator:jwt';
-                this.get('session')
-                  .authenticate(authenticator, credentials)
-                  .then(async() => {
-                    const tokenPayload = this.get('authManager').getTokenPayload();
-                    if (tokenPayload) {
-                      this.get('authManager').persistCurrentUser(
-                        await this.get('store').findRecord('user', tokenPayload.identity)
-                      );
-                    }
-                  });
-              });
-          });
+          this.get('loader').load('/auth/oauth/facebook')
+            .then(async response => {
+              window.location.replace(response.url);
```

Comment:
use ember methods.

Reply:
@CosmicCoder96 This being an external url cannot be redirected using transitionTo.

Reply:
@CosmicCoder96 If there is any other method for this, can you please guide me through it.

Reply:
@CosmicCoder96 Also, stripe is using torii, and I am not sure if it is required there or not, so I don't think we should uninstall it completely. I have deleted the torii functionality for facebook though. So can you please review the changes again.